core/modules/utils/dom/dragndrop.js

```diff
@@ -54,17 +54,28 @@ exports.makeDraggable = function(options) {
 			}
 			// Shift key alters the title-string by user-defined prefix and suffix
 			if (event.shiftKey && !event.ctrlKey) {
-                    		var dragShiftTiddler = $tw.wiki.getTiddler("$:/config/DragLinkShift");
-                		if(dragShiftTiddler.fields["prefix"] !== undefined && dragShiftTiddler.fields["suffix"] !== undefined) {
-                        		var titleStringPrefix = dragShiftTiddler.fields["prefix"];
-                        		var titleStringSuffix = dragShiftTiddler.fields["suffix"];
-                        		if(/^\$:\/.*/.test(titleString)) {
-                                		titleString = titleStringPrefix + titleString + titleStringSuffix;
-                        		} else {
-                                		titleString = titleStringPrefix + titleString.replace(/\[/g, '').replace(/]/g, '') + titleStringSuffix;
-                        		}
+			  var dragShiftTiddler = $tw.wiki.getTiddler("$:/config/DragLinkShift");
+			  if(dragShiftTiddler.fields["prefix"] !== undefined && dragShiftTiddler.fields["suffix"] !== undefined) {
```

Comment:
imo it will throw an error here if `dragShiftTiddler` is undefined

Reply:
yes you're totally right

Reply:
sorry for my indenting mess, I've mixed editors here